package.json

```diff
@@ -140,6 +140,7 @@
     "libp2p-tcp": "~0.13.0",
     "libp2p-webrtc-star": "~0.15.5",
     "libp2p-websocket-star": "~0.10.0",
```

Comment:
Can `libp2p-websocket-star` be removed?

Reply:
Depends. This forces ws-star-multi to use specifically that version because ws-star-multi has a `^` requirement instead of a `~`. So I'd leave it here just to make the version more specific

Reply:
I think ~ and ^ have the same semantics for pre 1.0 dependencies.

Reply:
> Um, but why [#1793 (comment)](https://github.com/ipfs/js-ipfs/pull/1793#discussion_r249005091) then, though?

It's in our commit linter for CI and is in our [JS code guidelines](https://github.com/ipfs/community/blob/master/CONTRIBUTING_JS.md#dependency-versions).

> AFAIK `~` matches same-minor and same-major, while `~` matches same-major only.

`^0.2.3 := >=0.2.3 <0.3.0` from https://www.npmjs.com/package/semver#caret-ranges-123-025-004 - `^` has a different meaning for 0.x versions (which just so happens to be the same as `~`).

i.e. unless there's another reason to keep it, it can be removed - it'll have no bearing on what version `libp2p-websocket-star-multi` uses as they have equivalent version ranges.

Reply:
> AFAIK ~ matches same-minor and same-major, while ~ matches same-major only.

:man_facepalming: one of them should be `^`, but seems wrong anyways

> i.e. unless there's another reason to keep it, it can be removed - it'll have no bearing on what version libp2p-websocket-star-multi uses as they have equivalent version ranges.

Agree